content/doc/book/managing/system-properties.adoc

```diff
@@ -1778,12 +1778,14 @@ properties:
 - name: jenkins.security.s2m.CallableDirectionChecker.allowAnyRole
   tags:
   - security
+  - obsolete
   def: |
     `true`
   since: 2.319 and 2.303.3
   description: |
     This flag can be set to `false` to explicitly reject `Callable` implementations that do not declare any required role.
-    As of early November 2021, it is unclear whether this can safely be set to `false`, or whether that would cause problems with some remoting built-in callables.
+    It is unclear whether this can safely be set to `false` in Jenkins before TODO, or whether that would cause problems with some remoting built-in callables.
+    This flag was removed in Jenkins 2.TODO.
```

Comment:
```suggestion
    It is unclear whether this can safely be set to `false` in Jenkins before 2.335, or whether that would cause problems with some remoting built-in callables.
    This flag was removed in Jenkins 2.335.
```